helm_deploy/prison-api/values.yaml

```diff
@@ -10,11 +10,10 @@ generic-service:
     tag: latest
     port: 8080
 
-  service:
```

Comment:
bit of tidying up now we don't deploy to azure as well.  Configuration moved here instead of each of the `values-*.yaml` files